### PR TITLE
Project Duplicate Problem fixing

### PR DESCRIPTION
**Pull request by nadilhassan**
When duplicating project, the duplicate is happening but the project tasks related to that particular project is not copying . instead of that some other tasks are copying to duplicated project.  I found that duplicateid is parsed to Save.php but Save.ph is looking for relate_id which is a bug.

## How To Test This

Try duplicating a project , it does not duplicate project tasks

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [ ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Project/Save.php [hunk 1]
```diff
@@ -133,4 +133,4 @@
 	$_REQUEST['return_url'] = "index.php?module=Project&action=view_GanttChart&project_id=" . $return_id;
     handleRedirect($return_id,'Project');
 }
-?>
\ No newline at end of file
+?>
```
Comment (daniel-samson) [inline modules/Project/Save.php]: PSR-2: Please remove ending ?>.

Comment (nadilhassan) [inline modules/Project/Save.php]: I have removed the new line....can you check that please?

Comment (daniel-samson) [inline modules/Project/Save.php]: @nadilhassan I can't see your change. When I cloned your repository the ?> is still in the file. Can you please fix your master branch?

Comment (daniel-samson) [inline modules/Project/Save.php]: Please read the guide on contributing to suitecrm. https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM

It is best to create a branch from the develop branch for features. Or create a branch from hotfix for issues / fixes. I this case you should create a branch from hotfix.

Comment (nadilhassan) [inline modules/Project/Save.php]: Thanks for you guide... I tried to remove that line and it is removing when I commit but after that it shows the same line ...I dont know why its happening like that. Meanwhile I have commited with my another account. Please merge that https://github.com/nadiln/SuiteCRM/pull/1/commits/04e99d12fc8a576d8c9d1fbde8c1ab2cd936c486

**Comment by daniel-samson** [comment]
@nadilhassan can you please close this pull request? as i have reviewed #3571.

**Comment by nadilhassan** [comment]
OK Sure.

**Comment by daniel-samson** [comment]
:+1:

**Pull request closed without merging**